### PR TITLE
Add a script for pulling the code from upstream

### PR DESCRIPTION
**Pull request by alvinypyim**
Might be an overkill :worried:

See what you think!

### CODE REVIEW

File: bin/fetch [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+mkdir -p tmp
+cd tmp
+
+TMP_FILE=$(mktemp)
+curl -o"$TMP_FILE" "$URL"
+unzip "$TMP_FILE"
+rm -v "$TMP_FILE"
+
+echo Remove the unnecessary execute bit of the distributed files.
+find . -type f -exec chmod -x {} ';'
```
Comment (msmirnoff) [inline bin/fetch:28]: Isn't this unnecessary?

And if we intend this, then why on the current dir instead of the unzipped result?

Comment (alvinypyim) [inline bin/fetch:28]: Because the zip file has no top level directories.

Comment (msmirnoff) [inline bin/fetch:28]: Ah yes, tmp containts TMP_FILE made by mktemp.

**Comment by msmirnoff** [comment]
:+1: even thought this is just a shambling horror.

**Pull request merged**